ios/Classes/AdmobRewardPlugin.swift

```diff
@@ -88,10 +88,17 @@ public class AdmobRewardPlugin: NSObject, FlutterPlugin {
         }
     }
     
-    private func loadRewardBasedVideoAd(id: Int, rewardBasedVideoAdUnitId: String) {
+    private func loadRewardBasedVideoAd(id: Int, rewardBasedVideoAdUnitId: String, nonPersonalizedAds: Bool) {
         let video = GADRewardedAd(adUnitID: rewardBasedVideoAdUnitId)
         rewardAds[id] = video
         let request = GADRequest()
+
+        if (nonPersonalizedAds) {
+            let extras = GADExtras()
+            extras.additionalParameters = ["npa": "1"]
```

Comment:
Can you please share a link to `npa` additional parameters docs.

I'm only seeing [Targeting](https://developers.google.com/admob/ios/targeting). It states 

> This parameter disables personalized advertising, including remarketing, for that specific ad request. It also disables requests to third-party ad vendors, such as ad measurement pixels and third-party ad servers.

Reply:
Sure, [here](https://developers.google.com/admob/android/eu-consent#forward-consent)'s where I found it.

Reply:
This approach is marked as deprecated. Aaah, now I think I'm connecting dots. This [iOS](https://developers.google.com/admob/ump/ios/quick-start), [Android](https://developers.google.com/admob/ump/android/quick-start) is the new EU consent method. This appears to be a different framework than what `admob_consent` is using.

Reply:
Oh yes, as you said, the UMP SDK seems to be the new recommended way to handle user consent (although it seems to be [a bit messy](https://stackoverflow.com/questions/63464270/how-to-request-non-personalized-admob-ads-with-the-new-unified-messaging-platfor)). But the `npa` parameter is, and [will still be](https://groups.google.com/g/google-admob-ads-sdk/c/8hz_x8cIOzQ/m/IxKuBtcmAAAJ) supported for a long time.